src/Elmish.WPF/ViewModel.fs

```diff
@@ -11,34 +11,6 @@ open Microsoft.Extensions.Logging
 open Elmish
 
 
-type LoggingViewModelArgs =
-  { performanceLogThresholdMs: int
-    log: ILogger
-    logPerformance: ILogger
-    nameChain: string }
-
-module LoggingViewModelArgs =
-
-  let getNameChainFor nameChain name =
-    sprintf "%s.%s" nameChain name
-
-  let getNameChainForItem nameChain collectionBindingName itemId =
-    sprintf "%s.%s.%s" nameChain collectionBindingName itemId
-
-  let map nameChain v = { v with nameChain = nameChain }
-
-type ViewModelArgs<'model, 'msg> =
-  { initialModel: 'model
-    dispatch: 'msg -> unit
-    loggingArgs: LoggingViewModelArgs }
-
-module ViewModelArgs =
-  let create initialModel dispatch nameChain loggingArgs =
-    { initialModel = initialModel
-      dispatch = dispatch
-      loggingArgs = LoggingViewModelArgs.map nameChain loggingArgs }
-
-
```

Comment:
I didn't notice this in the previous commit, but now I see that there types and functions are included in the public API.  Is that intention?

Reply:
Yes as the top level exposure eventually will be to supply create functions with dispatch and initial model arguments. That's why I want the helpers off in their own thing.

Reply:
But they don't need to be public yet, right?  If not, I would prefer to keep them internal. Then when they can be made public in the same commit that makes use of that capability.

Reply:
They actually do need to be made public at the end of this PR because `ViewModel` needs to public

Reply:
If you want, we can stop after the first commit ("Replace bindings with Create/Update/Get ViewModels") with this PR, since only the modification in the second commit requires this to be made public.

Reply:
[The first commit](https://github.com/elmish/Elmish.WPF/pull/449/commits/fe90ffc201a688e9494fc7aa701f9cc98e03bc02) is good.  If you want to split that off into its own commit, then I merge it now.  I will start reviewing [the second commit](https://github.com/elmish/Elmish.WPF/pull/449/commits/0fe347b65c927619abce9615f8ce5a310a219a1c) now.

Reply:
#453 @TysonMN created